extensions/sequelize/LICENSE

```diff
@@ -0,0 +1,25 @@
+Copyright (c) IBM Corp. and LoopBack contributors 2022.
```

Comment:
I think you should remove IBM Corp.

Reply:
also from all the files including the ones in `src` right?

Reply:
yes. any newly created files do not need to have `IBM Corp.` in the header.